pkgs/development/python-modules/luddite/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytestCheckHook
+, pytest-cov
+, pytest-socket
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "luddite";
+  version = "1.0.1";
```

Comment:
```suggestion
  version = "unstable-2021-07-16";
```

Reply:
The upstream maintainer has released a v1.0.2 with the necessary patches. The version is now stable again